src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2297,3 +2297,57 @@
   - name: List all versions of parent template spec.
     text: az ts list -g MyResourceGroup -n TemplateSpecName
 """
+
+helps['bicep'] = """
+type: group
+short-summary: Bicep CLI command group.
+"""
+
+helps['bicep install'] = """
+type: command
+short-summary: Install Bicep CLI.
+examples:
+  - name: Install Bicep CLI.
+    text: az bicep install
+  - name: Install a specific version of Bicep CLI.
+    text: az bicep install --version v0.2.212
+"""
+
+helps['bicep upgrade'] = """
+type: command
+short-summary: Upgrade Bicep CLI to the latest version.
+"""
+
+helps['bicep build'] = """
+type: command
+short-summary: Build one or more Bicep files.
+examples:
+  - name: Build a Bicep file.
+    text: az bicep build --files <bicep_file>
```

Comment:
How about using `--files {bicep-file}` to represent the place holder of file, so that the format can be consistent with `--template-file {template-file}`

```suggestion
    text: az bicep build --files {bicep-file}
```